### ISSUE TITLE
Some new marker to represent connections/disconnections/when SQL logs are being loaded

### ISSUE THREAD

**Issue by MaxLeiter** [thelounge/thelounge#2300]
Connections/disconnections in topic refer to Lounge instance disconnecting from network.

When you restart Lounge, it can be hard to tell where you disconnected (logs are being loaded) and where you just joined.

**Comment by astorije** [comment]
I'm not sure we should do that actually. Users shouldn't have to be aware that the remote server restarted. However, showing that they were disconnected/reconnected or left/re-joined the channel, yes, that'd be a good thing.

**Comment by MaxLeiter** [comment]
yeah @astorije that's what I meant -- when logs are reloaded (which is kind of the same as if the server restarts, but also for channel parts/joins, i guess)

**Comment by xPaw** [comment]
Don't you already see joins/topic when the server restarts and it rejoins the channels?

If we don't want a separate "log loaded" event, this issue is effectively solved?

**Comment by McInkay** [comment]
Sure, but you can also see all of those at other times, and it's hard to tell the difference between a disconnect/reconnect and a netsplit. You do see yourself leaving and joining, which is the only time you see it, but that definitely doesn't stand out and would be really hard to see amongst all the other messages.
